src/ingest_validation_tools/directory-schemas/codex.yaml

```diff
@@ -17,50 +10,62 @@
   description: 'PDF export of Powerpoint slide deck containing the Image Analysis Report'
   is_qa_qc: True
 -
-  pattern: 'drv_[^/]+/channelNames\.txt'
-  description: 'Text file produced by the Akoya software which contains the (linearized) channel number and the Name/ID/Target of the channel (required for HuBMAP pipeline)'
+  pattern: '(raw|src_.*)/.*'
+  description: 'Raw data directory.'
   required: True
 -
-  pattern: 'src_[^/]+/experiment\.json'
-  description: 'JSON file produced by the Akoya software which contains the metadata for the experiment, including the software version used, microscope parameters, channel names, pixel dimensions, etc. (required for HuBMAP pipeline)'
+  pattern: '(processed|drv_.*)/.*'
+  description: 'Processed files produced by the Akoya software or alternative software.'
   required: True
 -
-  pattern: 'drv_[^/]+/experiment\.json'
-  description: 'JSON file produced by the Akoya software which contains the metadata for the experiment, including the software version used, microscope parameters, channel names, pixel dimensions, etc. (required for HuBMAP pipeline)'
+  pattern: '(raw|src_.*)+exposure_times\.txt'
+  description: 'Comma separated text file used for background subtraction that contains valid exposure times for all cycles [e.g: Cycle,CH1,CH2,CH3,CH4].'
```

Comment:
Suggest moving the more precise patterns above the loose directory patterns: The file descriptions in the portal are determined by the first pattern that matches. (I think.)

Reply:
@mccalluc the difference is that `src_[^/]` won't accept `/` as a first character, correct?

Reply:
- `src_.+` would accept `src_/`
- `src_[^/]+` would not.

`.` is any character (except newline, but don't worry about that).
`[^/]` is any character expect `/`.